community/libmustache/APKBUILD

```diff
@@ -0,0 +1,39 @@
+# Contributor: Renoir Boulanger <hello@renoirboulanger.com>
+# Maintainer: John Boehr <jbboehr@gmail.com>
```

Comment:
I wasn't sure about the meaning in the context of that annotation.
I wanted to leave mark about the author of the underlining code and not take credit.

What's the meaning of Contributor and Maintainer here then?

I thought the maintainer was the original underlying code author and contributor the one who's bringing it into Alpine build package system.